tests/e2e/auth.setup.ts

```diff
@@ -8,6 +9,8 @@ import { authFile } from './fixtures/helpers';
 // msedge/firefox, as the 'setup' project doesn't know which project we're
 // running with when using `--project` on CLI.
 setup('authenticate', async ({ page }) => {
+  setup.skip(existsSync(authFile), 'Already authenticated');
```

Comment:
`authFile` is deleted when running end-to-end tests first time. If we're watching, or running individual tests via UI, we save time by skipping this setup each time (all tests depend on `setup`).